nuget/MobileCenter.nuspec

```diff
@@ -21,6 +21,11 @@
         <dependency id="Newtonsoft.Json" version="6.0.1"/>
         <dependency id="sqlite-net-pcl" version="1.3.1"/>
       </group>
+      <!-- Use dependency groups for iOS and Android so that NETStandard dependencies aren't added to them -->
```

Comment:
update comment so that it's not .net standard specific and so we can merge even with the rollback